hassio/src/components/hassio-upload-backup.ts

```diff
@@ -32,13 +36,26 @@ export class HassioUploadBackup extends LitElement {
         .uploading=${this._uploading}
         .icon=${mdiFolderUpload}
         accept="application/x-tar"
-        label="Upload backup"
-        supports="Supports .TAR files"
+        .label=${this.localize?.(
+          "ui.panel.page-onboarding.restore.upload_backup"
```

Comment:
We can not use `onboarding` translations in a `hassio` component, make sure it works for both then, by having the parent set an optional translation key.

Reply:
I can but is it worth the effort? Hassio backup is replaced by the new backup dashboard. Because of that I kept the static english text.